code/src/java/pcgen/system/ConfigurationSettings.java

```diff
@@ -90,19 +85,6 @@ public static void setCountry(String country)
 		setSystemProperty(USER_COUNTRY, country);
 	}
 
-	/**
-	 * @return the current user directory
-	 */
-	public static String getUserDir()
-	{
-		return SystemUtils.USER_DIR;
-	}
-
-	public static ConfigurationSettings getInstance()
```

Comment:
I don't  get this - why spread lots of nulls rather than having a convenience method?  What's the logic here?